### PR TITLE
i18n(es): Translated `node.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
 Translated `node.mdx` to spanish ✅

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/node.mdx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+app.use(ssrHandler);
+
+app.listen({ port: 8080 });
+```
+
+Ten en cuenta que el modo middleware no sirve de archivos. Necesitarás configurar tu framework HTTP para que lo haga por ti. De forma predeterminada, los activos del cliente se escriben en `./dist/client/`.
+
+### Standalone
+
+En modo standalone, un servidor inicia cuando el punto de entrada del servidor se ejecuta. De forma predeterminada se construye a `./dist/server/entry.mjs`. Puedes ejecutarlo con:
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/node.mdx:127]:
```suggestion
En el modo standalone, un servidor se inicia cuando el punto de entrada del servidor se ejecuta. De forma predeterminada se construye a `./dist/server/entry.mjs`. Puedes ejecutarlo con:
```

File: src/content/docs/es/guides/integrations-guide/node.mdx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    import node from '@astrojs/node';
+
+    export default defineConfig({
+      output: 'server',
+      adapter: node({
+        mode: 'middleware'
+      }),
+    });
+    ```
+*   El modo `standalone` permite construir un servidor que inicia automáticamente cuando el módulo de entrada se ejecuta. Esto te permite desplegar tu construcción a un host sin ningún código adicional.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/node.mdx:79]:
```suggestion
*   El modo `standalone` permite construir un servidor que se inicia automáticamente cuando el módulo de entrada se ejecuta. Esto te permite desplegar tu construcción a un host sin ningún código adicional.
```

File: src/content/docs/es/guides/integrations-guide/node.mdx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+## Configuración
+
+@astrojs/node puede ser configurado pasando opciones al adaptador. Las siguientes opciones están disponibles:
+
+### Modo
+
+Controla si el adaptador construye en modo `middleware` o `standalone`.
+
+*   El modo `middleware` permite el uso de la salida construida como middleware para otro servidor de Node.js, como Express.js o Fastify.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/node.mdx:67]:
```suggestion
*   El modo `middleware` permite que la salida generada se utilice como middleware para otro servidor de Node.js, como Express.js o Fastify.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Yan-Thomas — APPROVED** [review]
Good job peeps, LGTM!

**Pull request merged**